### PR TITLE
Change OTC authorization to Bearer [#168661200]

### PR DESCRIPTION
**Pull request by vitaliypodlypnyi**

### CODE REVIEW

File: CredentialPostTest/CredentialPostTest/appsettings.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "AppUrl": "https://app.zwapgrid.com/",
     "PartnerToken": "{yourToken}",
     "ClientId": "{yourClientId}",
-    "ClientSecret": "{yourClientSecret}"
+    "ClientSecret": "{yourClientSecret}",
+    "AccessToken": "{yourAccessToken}"
```
Comment (dennisahlin) [inline CredentialPostTest/CredentialPostTest/appsettings.json:19]:
```suggestion
    "AccessToken": "{accessTokenOfTenant}"
```

**Comment by dennisahlin** [comment]
Wait, access token? Shouldn't it be on the user, like `user.CompanyAccessToken`, not in settings?

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs [hunk 2]
```diff
@@ -50,35 +55,35 @@ public async Task OnGet()
                 return;
           
             //Fetch user
-            var user = await _userManager.GetUserAsync(User);
-            
+            _user = await _userManager.GetUserAsync(User);
```
Comment (dennisahlin) [inline CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:58]: Noot sure about moving this to a field... What if result is null, what if more functions are added (this is an index page so quite possible)?

Comment (vitaliypodlypnyi) [reply CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:58]: you suppose throwing this guy from function to function? 😄

Comment (dennisahlin) [reply CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:58]: Yeah, I do. But I'm not sure... @KaterynaZorina ?

Comment (vitaliypodlypnyi) [reply CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:58]: Decided to add null check right after user setting

**Review by dennisahlin — APPROVED** [review]

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs (hunk 2 above)
Comment (KaterynaZorina) [reply CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:58]: @dennisahlin private field with null check looks ok for me. We can't guarantee that user won't pass `null` as a method parameter(unless we will be checking for null every time). `OnGet()` method will always be called, so we can rely on it.

**Review by KaterynaZorina — APPROVED** [review]

**Pull request merged**